gammapy/background/off_data_background_maker.py

```diff
@@ -198,17 +198,17 @@ def filename(self, modeltype, group_id, smooth=False):
         else:
             return 'background_{}_group_{:03d}_table.fits.gz'.format(modeltype, group_id)
 
-    def save_model(self, modeltype, ngroup):
+    def save_model(self, modeltype, ngroup, smooth=False):
```

Comment:
I'm not sure it's a good idea to change the filename depending on smooth / no-smooth.

This class will get feature-richer, there will be different ways to smooth the model.
Probably filename management should be left up to the user in those cases?
(but leave as-is if you like for now)